src/Search/FluentSearchVariant.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace TractorCow\Fluent\Search;
+
+use TractorCow\Fluent\Extension\FluentExtension;
+use TractorCow\Fluent\State\FluentState;
+use TractorCow\Fluent\Model\Locale;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\FullTextSearch\Search\Variants\SearchVariant;
+use SilverStripe\FullTextSearch\Search\SearchIntrospection;
+use SilverStripe\FullTextSearch\Search\Queries\SearchQuery;
+
+class FluentSearchVariant extends SearchVariant
```

Comment:
We need a check here to see if `SearchVariant` exists before declaring the class. It's not ideal but this will break without FTS installed otherwise